README.md

```diff
@@ -30,7 +30,7 @@ If you'd like to join the community and help improve these tools, great!
 1. Review our [Contributor Guidelines](/CONTRIBUTING.md) and [Code of Conduct](/CONDUCT.md)
 1. Jump on the [Archivers chat (archivers.slack.com)](https://archivers.slack.com/), anyone can request an invite from [archivers-slack.herokuapp.com slackin](https://archivers-slack.herokuapp.com/)
     - Introduce yourself in `#introductions`, key starting places for conversations are `#general`, `#dev` and `#community-building`
-    - Ping one of the EDGI coordinators (@dcwalk, @patcon, or @mattprice) with your GitHub name to be added to the organization
+    - Ping one of the EDGI coordinators (@lightandluck, @kelsey, or @mattprice) with your GitHub name to be added to the organization
```

Comment:
I think this should link to @Frijol rather than `@kelsey`?

Also I'd cut `@mattprice` (should have been @titaniumbones anyway)

Reply:
That step was for getting in touch on Archivers. That did prompt me to add getting in touch with us on Github as well. 

Also removed Matt as contact and changed #dev to #datatogether for the key starting points in Slack. 

How's that look?